scripts/analyze_all_merchants.py

```
@@ -0,0 +1,463 @@
+    print(
+        f"Merchants with word data: {merchants_with_data} ({100*merchants_with_data/total_merchants:.1f}%)"
+    )
+    print(f"Merchants with errors: {merchants_with_errors}")
+
+    if run_llm:
+        # Receipt type distribution
+        itemized = [r for r in results if r.receipt_type == "itemized"]
+        service = [r for r in results if r.receipt_type == "service"]
+        llm_errors = [r for r in results if r.llm_error]
+
+        print(f"\n=== RECEIPT TYPE CLASSIFICATION ===")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)